symfony/src/Helper/WorkQueue.php

```diff
@@ -634,6 +636,8 @@ public static function getExportHeaders()
         $headers[] = 'COPE Feb PPI Survey Complete';
         $headers[] = 'COPE Feb PPI Survey Completion Date';
         $headers[] = 'Core Participant Minus PM Date';
+        $headers[] = 'Summer Minute Survey Complete';
+        $headers[] = 'Summer Minute Survey Completion Date';
```

Comment:
This is slightly inconsistent from the other survey column headers ("Survey" instead of "PPI Survey"). I added a [comment](https://precisionmedicineinitiative.atlassian.net/browse/HPRO-823?focusedCommentId=88415) to the JIRA ticket to get clarification.